data/basetest/problem/problem_data.json

```diff
@@ -24,16 +24,20 @@
         {
             "Network_problem":
             {
-                "StochasticPQnode_data": {
-                    "number_of_stochastic_steps": 1000,
-                    "theta_P": 0.3,
-                    "sigma_P": 0.0,
-                    "theta_Q": 0.3,
-                    "sigma_Q": 0.0
+                "defaults" : {
+                    "StochasticPQnode": {
+                        "number_of_stochastic_steps": 1000,
+                        "theta_P": 0.3,
+                        "sigma_P": 0.0,
+                        "theta_Q": 0.3,
+                        "sigma_Q": 0.0
+                    },
+                    "Pipe": {
+                        "balancelaw": "Isothermaleulerequation",
+                        "scheme": "Implicitboxscheme",
+                        "desired_delta_x": 10000
+                    }
```

Comment:
This demonstrates how defaults would be defined

Reply:
nice, exactly what I wanted!